front/app/modules/commercial/insights/admin/containers/Insights/Edit/index.tsx

```diff
@@ -298,7 +301,8 @@ const EditInsightsView = ({
                     bgHoverColor={darken(0.05, colors.lightGreyishBlue)}
                     onClick={selectCategory(category.id)}
                   >
-                    {category.attributes.name}
+                    <span>{truncate(category.attributes.name, 20)}</span>
```

Comment:
Definitely fine for now, but we should migrate these truncate calls towards a css solution (`text-overflow: ellipsis`) that'd be more appropriate for accessibility and robustness.

Reply:
Nice, thanks for pointing this out! Changed it.